frontend/src/features/map/tools/vessel_visibility/LastPositionsSlider.tsx

```diff
@@ -1,8 +1,9 @@
-import Nouislider from 'nouislider-react'
 import { useEffect, useState } from 'react'
 import { RangeSlider } from 'rsuite'
 import styled from 'styled-components'
 
+import Nouislider from '../../../../ui/Nouislider'
```

Comment:
Why copy this library ? This is well maintained

Reply:
Talked in chat.